packages/jupyter-ai/src/components/settings/model-fields.tsx

```diff
@@ -13,9 +13,31 @@ export type ModelFieldProps = {
 };
 
 export function ModelField(props: ModelFieldProps): JSX.Element {
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
   function handleChange(
-    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+    format: string
```

Comment:
Why does `format` need to be passed as an argument to `handleChange`? Can't `handleChange` read `props.field.format` from its outer scope?